dashboard/src/modules/components/AuthComponent/index.less

```diff
@@ -96,16 +96,4 @@
         background: transparent; /* make scrollbar transparent */
     }
 }
-.login-wrapper {
-    top: 10%;
-    .login-right-component {
-        top: 15%;
-    }
-}
-.signup-wrapper {
-    top: 5%;
 
-    .login-right-component {
-        top: 10%;
-    }
-}
```

Comment:
Now that you've removed `dashboard/src/modules/components/AuthComponent/index.jsx`, I'm wondering whether `dashboard/src/modules/components/AuthComponent/index.less` should be renamed.

Reply:
It can be index.less as it has styles for other child components